src/AddIns/Uno.UI.MediaPlayer.Skia.X11/SharedMediaPlayerExtension.cs

```diff
@@ -135,10 +129,46 @@ private set
 	// On Win32, EnableMouseInput needs to be false, or else libvlc will capture the pointer and we won't receive
 	// any mouse events. Attempting to do this later below doesn't work for some reason. It needs to be
 	// right after constructing the LibVLCSharp.Shared.MediaPlayer
-	internal LibVLCSharp.Shared.MediaPlayer VlcPlayer { get; } = new LibVLCSharp.Shared.MediaPlayer(_vlc) { EnableMouseInput = false, EnableKeyInput = false };
+	internal LibVLCSharp.Shared.MediaPlayer VlcPlayer { get; }
+
+	public static void PreloadVlc()
+	{
+		Task.Run(() =>
```

Comment:
Why is the logic wrapped in Task.Run?
I don't immediately see why. It would be good to add a comment anyways.

Reply:
The idea is to preload VLC without blocking the UI thread.